app/tracing.ts

```diff
@@ -0,0 +1,227 @@
+import { BatchSpanProcessor, SamplingDecision, SamplingResult } from '@opentelemetry/sdk-trace-base';
```

Comment:
I was thinking at the top of files like these it would be useful to add a one or two sentence explanation of its purpose.

Reply:
Done.